src/harness/fake_sas.py

```diff
@@ -222,6 +237,18 @@ def do_POST(self):
     self.end_headers()
     self.wfile.write(json.dumps(response))
 
+  def do_GET(self):
+    """Handles GET requests."""
+    path, value = self._parseUrl(self.path)
+    if path == "/v1.0/sas_impl":
```

Comment:
Do we really need to include the "/v1.0" part? The POST version doesn't have it.